src/main.rs

```diff
@@ -23,15 +24,42 @@ const PLATFORM: &'static [PlatformData] = &[
 fn main() {
     let mut col = Collector::new();
 
+	let matches = App::new("conga")
+        .version("0.1.0")
+        .author("")
+        .about("System Configurator Gatherer")
```

Comment:
Is configurator a word? Maybe configuration is more clear?

Reply:
sure isn't, good catch. will change it to match the repo description